bin/jest.js

```diff
@@ -113,6 +117,22 @@ var argv = optimist
       ),
       type: 'boolean'
     },
+    watch: {
+      alias: 'm',
+      description: _wrapDesc(
+        'Run all tests and then watch files in your testPathDirs for ' +
+        'changes and then rerun tests related to changed files and ' +
+        'directories.'
+      ),
+      type: 'boolean'
+    },
+    skipFirstRun: {
```

Comment:
not really happy with this option. Can we make `watch` take arguments? Like `--watch=skip`?